### PR TITLE
Move TryGetPlatformHandle to TopLevel

### PR DESCRIPTION
**Pull request by fitdev**
## What does the pull request do?
As per #11454 moves `TryGetPlatformHandle` down the hierarchy to `TopLevel`

## Breaking changes
None

## Fixed issues
Partially addresses #11454

### CODE REVIEW

**Comment by fitdev** [comment]
@dotnet-policy-service agree

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0035186-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

File: src/Avalonia.Controls/TopLevel.cs [hunk 1]
```diff
@@ -376,6 +376,15 @@ public ILayoutManager LayoutManager
...
         public ITopLevelImpl? PlatformImpl { get; private set; }
         
+        /// <summary>
+        /// Trys to get the platform handle for the TopLevel-derived control.
+        /// </summary>
+        /// <returns>
+        /// An <see cref="IPlatformHandle"/> describing the window handle, or null if the handle
+        /// could not be retrieved.
+        /// </returns>
+	public IPlatformHandle? TryGetPlatformHandle() => ((IWindowBaseImpl?) PlatformImpl)?.Handle;
```
Comment (grokys) [inline src/Avalonia.Controls/TopLevel.cs:386]: Indentation looks wrong here?

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0035225-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**